package.json

```diff
@@ -54,6 +54,7 @@
     "bull-board": "0.5.0",
     "cookie-parser": "1.4.4",
     "cors": "2.8.5",
+    "crypto": "1.0.1",
```

Comment:
You can remove this, it's part of node itself now, see https://www.npmjs.com/package/crypto.

Reply:
Removed `crypto`.